pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -213,6 +215,7 @@ pub struct DeltaLayerInner {
     // values copied from summary
     index_start_blk: u32,
     index_root_blk: u32,
+    vectored_blob_reader: VectoredBlobReader,
```

Comment:
Same comment as for image layer: if we can avoid making the layer struct bigger that would be a win.

Reply:
https://github.com/neondatabase/neon/pull/6780/commits/0804e0c776b12e841861d53e42f646b5ce92a17d